webpages/settings/index.js

```diff
@@ -119,7 +119,7 @@ const ResetDropdown = Vue.extend({
 });
 Vue.component("reset-dropdown", ResetDropdown);
 const AddonBody = Vue.extend({
-  props: ["addon"],
+  props: ["addon", "category"],
```

Comment:
Why does the reset dropdown need to know about the category...?

Reply:
bruh plz read